src/AutoRest.CSharp/Common/Output/Expressions/Azure/AzureExtensibleSnippets.AzureRestOperationsSnippets.cs

```diff
@@ -37,9 +38,13 @@ public override TypedValueExpression GetTypedResponseFromEnum(EnumType enumType,
                 return ResponseExpression.FromValue(EnumExpression.ToEnum(enumType, rawResponse.Content.ToObjectFromJson(typeof(string))), rawResponse);
             }
 
-            public override TypedValueExpression GetTypedResponseFromBinaryDate(Type responseType, TypedValueExpression response)
+            public override TypedValueExpression GetTypedResponseFromBinaryDate(Type responseType, TypedValueExpression response, string? contentType = null)
```

Comment:
here is a typo in the method name
```suggestion
            public override TypedValueExpression GetTypedResponseFromBinaryData(Type responseType, TypedValueExpression response, string? contentType = null)
```